climate_emotions_map/layout.py

```diff
@@ -153,25 +219,31 @@ def create_header():
         children=[
             dmc.Stack(
                 justify="center",
-                h=70,
+                h=HEADER_HEIGHT,
                 children=dmc.Grid(
                     justify="space-between",
                     align="center",
                     children=[
                         dmc.GridCol(
-                            children=dmc.Anchor(
-                                "US Climate Emotions Map 2024",
-                                size="xl",
-                                href="/",
-                                underline=False,
+                            children=dmc.Stack(
+                                gap=5,
+                                justify="center",
+                                children=[
+                                    dmc.Anchor(
+                                        SECTION_TITLES["app"],
+                                        size="xl",
+                                        href="/",
+                                        underline=False,
+                                    ),
+                                    create_app_subtitle(),
+                                ],
                             ),
                             span="content",
                         ),
                         dmc.GridCol(
                             dmc.Group(
                                 justify="flex-end",
                                 # TODO: Add GitHub link? Not sure if needed/wanted.
-                                # TODO: Add link to paper
```

Comment:
Are we not going to link to the paper later?
Good idea to add the GitHub link to the repo if not we should remove the parent `dmc.GridCol` entirely since it has no content.